src/main/java/ari/tasks/DeadlineTask.java

```diff
@@ -0,0 +1,46 @@
+package ari.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Reprensents a Task with Deadline
+ */
+public class DeadlineTask extends Task {
+    protected String time;
+    protected LocalDate nowTime;
```

Comment:
Should a more intuitive variable name be used? Maybe something like deadlineDate